MarkdownKit/Sources/Common/Elements/Header/MarkdownHeader.swift

```diff
@@ -33,12 +33,15 @@ open class MarkdownHeader: MarkdownLevelElement {
       attributedString.deleteCharacters(in: range)
   }
 
-    open func attributesForLevel(_ level: Int) -> [NSAttributedString.Key: AnyObject] {
+  // Modified by Xiaochen Guo (xiaochen.guo@accedo.tv)
+  // The `bold()` function might return nil as some libs have different naming convention
+  // Now allow preset header font
+  open func attributesForLevel(_ level: Int) -> [NSAttributedString.Key: AnyObject] {
     var attributes = self.attributes
     if let font = font {
         let headerFontSize: CGFloat = font.pointSize + 4 + (-1 * CGFloat(level) * CGFloat(fontIncrease))
-      
-      attributes[NSAttributedString.Key.font] = font.withSize(headerFontSize).bold()
+        let targetFont = font.withSize((headerFontSize > font.pointSize) ? headerFontSize : font.pointSize)
```

Comment:
didnt understand this line . headerFontSize is calculated above shouldnt we just use this size not sure why headerFontSize was even calculated maybe the library has some logic behind it  . 

 `let targetFont = font.withSize(headerFontSize)`

Reply:
changed!